src/current/v23.2/install-cockroachdb-windows.md

```diff
@@ -15,85 +15,94 @@ docs_area: deploy
+{% include windows_warning.md %}
 
-Use one of the options below to install CockroachDB.
+Use one of the options below to install CockroachDB. To upgrade an existing cluster, refer to [Upgrade to {{ page.version.version }}]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}).
 
 <section id="download-the-binary-windows" markdown="1" class="install-option">
 <h2 id="install-binary">Download the executable</h2>
 
-  {% include windows_warning.md %}
-
-1. Using PowerShell, run the following script to download the [CockroachDB {{ page.release_info.version }} archive for Windows](https://binaries.cockroachdb.com/cockroach-{{ page.release_info.version }}.windows-6.2-amd64.zip) and copy the binary into your `PATH`:
-
-    {% include_cached copy-clipboard.html %}
-    ~~~ powershell
-    $ErrorActionPreference = "Stop"; [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12;$ProgressPreference = 'SilentlyContinue'; $null = New-Item -Type Directory -Force $env:appdata/cockroach; Invoke-WebRequest -Uri https://binaries.cockroachdb.com/cockroach-{{ page.release_info.version }}.windows-6.2-amd64.zip -OutFile cockroach.zip; Expand-Archive -Force -Path cockroach.zip; Copy-Item -Force "cockroach/cockroach-{{ page.release_info.version }}.windows-6.2-amd64/cockroach.exe" -Destination $env:appdata/cockroach; $Env:PATH += ";$env:appdata/cockroach"
-    ~~~
-
-    {{site.data.alerts.callout_success}}
-    To run a PowerShell script from a file, use syntax like `powershell.exe -Command "{path_to_script}"`.
-    {{site.data.alerts.end}}
-
-    We recommend adding `;$env:appdata/cockroach` to the `PATH` variable for your system environment so you can execute [cockroach commands](cockroach-commands.html) from any shell. See [Microsoft's environment variable documentation](https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_environment_variables#saving-changes-to-environment-variables) for more information.
-
-1. In PowerShell or the Windows terminal, check that the installation succeeded:
-
-    {% include_cached copy-clipboard.html %}
-    ~~~ shell
-    cockroach version
-    ~~~
-
-1. Keep up-to-date with CockroachDB releases and best practices:
-
-    {% include marketo-install.html uid="1" %}
+You can download and install CockroachDB for Windows in two ways. Either:
+
+- **Recommended**: Visit [Releases]({% link releases/index.md %}?filters=windows) to download CockroachDB. The archive contains the `cockroach.exe` binary. Extract the archive and optionally copy the `cockroach.exe` binary into your `PATH` so you can execute [cockroach commands]({% link {{ page.version.version }}/cockroach-commands.md %}) from any shell. Releases are rolled out gradually, so the latest version may not yet be downloadable.
+
+- Instead of downloading the binary directly, you can use PowerShell to download and install CockroachDB:
```

Comment:
non-blocking comment: if we're in here touching stuff, and this is not the recommended install instruction, why not rip it out instead of refactor it?

completely defer to your product area AOR and expertise of course!  I assume it's there b/c "someone asked for it"

I just wonder if the set intersection of "users who need to copy/paste Powershell to download a binary" vs. "users who should probably be downloading from the Releases page" is a null set 

happy to leave for another day but feel obligated to say

Reply:
It's because PowerShell gets them something that sorta feels like a service and this is the "Windows" way to do it. Windows isn't supported so this is just kind of gravy. I think we hope it's a pain to use this and they will do it the way we want, unless they really needed to script it.